browse/templates/login.html

```diff
@@ -0,0 +1,7 @@
+<div class="login">
```

Comment:
Could this be moved to auth?

Reply:
I'm not sure what you mean. The template, or something else?

Reply:
The template for the login could live in auth. So it would be consistent across browse, search, auth, mod, admin, labs, etc.